src/_data/books-dart3.yml

```diff
@@ -0,0 +1,8 @@
+- title: 'Dart Apprentice'
+  authors:
+    - Jonathan Sande
+    - Matt Galloway
+  publisher: raywenderlich
+  cover: dart-apprentice.png
```

Comment:
Could you update this image to match the latest on the book's page? Seems at least the banner has changed.

I believe the current photo is located at `src/_assets/image/cover/dart-apprentice.png`.

Reply:
I'm so sorry I didn't catch this, complete oversight on my part.

I've also incorporated all the other changes you mention below. So I'm resolving them instead of mentioning done on each one. (My reasoning being that of not wanting to add to your notifications for nothing 😅)